sympy/combinatorics/pc_groups.py

```diff
@@ -41,27 +48,112 @@ def is_prime_order(self):
                 return False
         return True
 
+    def is_consistent(self):
+        if self.power_exponents.sort() == self.relative_orders.sort():
```

Comment:
Is this right? Surely the order of `power_exponents` and `relative_orders` matters.